buyer-app/buyer-app/src/state/entities/downloadData/sagas.js

```diff
@@ -0,0 +1,46 @@
+import { put, takeLatest, all, call } from "redux-saga/effects";
+import download from "downloadjs";
+
+import * as NotificationActions from "state/entities/notifications/actions";
+import { getData } from "./helpers";
+
+import * as Actions from "./actions";
+
+/**
+ * Download data
+ * @yield {[type]} [description]
+ */
+function* downloadData(action) {
+  const { dataOrder } = action.payload;
+  try {
+    const response = yield call(getData, dataOrder.id);
+    if (response.error) {
+      yield put(
+        NotificationActions.createNotification({
+          message: response.message,
```

Comment:
From `getData`'s operation interface, I think this is:
```suggestion
          message: response.error.message,
```